### DISCUSSION TITLE
docker compose up extremely slow

### DISCUSSION THREAD

**Discussion by czhang96** [alshedivat/al-folio#2283] [Q&A] [answered]
On a fresh version of the repo, `docker compose up` is extremely slow (takes ~5 min). 
It seems that jekyll is hanging on this step specifically

`jekyll-1  | Post-Convert Hooks: assets/css/main.css.map`

If i add main.scss to the exludes, its much faster (but we lose all the styling)

**Comment by george-gca** [comment]
Are you using the slim version or the regular one?

@pourmand1376 any ideas on this?

**Reply by pourmand1376** [reply]
Hi. 

The slim version only differs in size. Nothing else should be different.

How much styling do we lose? Can you send an image to see the cost?

**Reply by george-gca** [reply, marked as answer]
`main.scss` is basically where all the styling of the site is loaded, so it can't be ignored.

I have one idea: try changing how both the font awesome and tabler icons are loaded. Comment their lines on `assets/css/main.scss` and add their import via cdn on `_includes/head.liquid`. See if that helps.

When building the `main.scss` we basically import the whole font awesome and tabler icons. This is actually better for 2 reasons:
- including files locally is faster than importing from a cdn
- when building for deploy, after `purgecss` does its job, we are left with only the icons that were actually used, so a lot of the css related to these fonts are discarded.
